src/main/java/uk/gov/hmcts/reform/hmc/api/services/CaseFlagDataServiceImpl.java

```diff
@@ -57,37 +75,44 @@ public CaseDetailResponse getCcdCaseData(CaseDetails caseDetails) throws IOExcep
     public void setCaseFlagData(ServiceHearingValues serviceHearingValues, CaseDetails caseDetails)
             throws IOException {
 
-        List<PartyFlagsModel> partiesFlagslList = new ArrayList<>();
+        List<PartyFlagsModel> partiesFlagsModelList = new ArrayList<>();
         List<PartyDetailsModel> partyDetailsModelList = new ArrayList<>();
         CaseDetailResponse ccdResponse = getCcdCaseData(caseDetails);
         setBaseLocation(serviceHearingValues, ccdResponse);
         List<Element<PartyDetails>> applicantLst = ccdResponse.getCaseData().getApplicants();
+        log.info("c100 appl list {}", applicantLst);
```

Comment:
either remove looger or change info to debug

Reply:
resolved